src/tutorials/Alphish/GM_Patterns__Step_Manager.md

```diff
@@ -0,0 +1,203 @@
+---
+title: GameMaker Patterns: Step Manager
+description: A tutorial describing a Step Manager pattern, for ensuring correct execution order and easily disabling part of gameplay logic.
+link: N/A
```

Comment:
I really need to get around to stripping out these automatically... But you don't need `link` at all, if there's no link to specify. The website will ignore any missing metadata.